frontend/ui/widget/confirmbox.lua

```diff
@@ -76,62 +76,73 @@ function ConfirmBox:init()
             self.key_events.Close = { { Device.input.group.Back } }
         end
     end
+
+    self.added_widgets_width = math.floor(math.min(Screen:getWidth(), Screen:getHeight()) * 2/3)
     local text_widget = TextBoxWidget:new{
         text = self.text,
         face = self.face,
-        width = math.floor(math.min(Screen:getWidth(), Screen:getHeight()) * 2/3),
+        width = self.added_widgets_width,
```

Comment:
As it applies also to this TextBoxWidget (which is not an "added widget"), better name that subwidget_width or inner_width (although I may have already used inner_width for some internal stuff in other widgets, so mine may be fetched on widget you don't expect it).

But ok, I see you expect it to be named like that in the 2 other widget this PR touches, where may be it make sense for it to be named like that.